examples/cpp/inference/infer_multiple_networks.cc

```diff
@@ -164,6 +164,11 @@ int main(int argc, char** argv) {
     return -1;
   }
 
+// Register cpu backend for static linking
```

Comment:
do we need to register here, since we have a registration in create backend?

Reply:
yeah it was failing otherwise because we are using set backend here

Reply:
TODO: Revisit this to see if we can remove registering here and register only in BackendManager.